### ISSUE TITLE
Do I understand the layers of my model right?

### ISSUE THREAD

**Issue by Yingyingzhang15** [fchollet/keras#2105]
Hi, I'm new to neural Keras, and I'm confused about the number of layers in my model.I want to have a model of three layers. 

First layer: 4 neurons
Second layer:5 neurons
Third layer: 3 neurons

My code is as follows:

model = Sequential()
model.add(Dense(5, input_dim=4))
model.add(Activation('relu'))
model.add(Dense(3))
model.add(Activation('softmax'))

Am I right?
Actually, I added two dense layers in the model, but it seems that the model is a three layer neural network. I'm so confused. Can you help me?Thank you very much!

**Comment by meanmee** [comment]
Right, In papers we have input layer(or many input layers) + many hidden layers + output layer(or many output layers). But in Keras, the input_dim or input_shape is likely to be the input layer.

**Comment by Yingyingzhang15** [comment]
@meanmee Thank you for your help!
